src/openms/include/OpenMS/FILTERING/DATAREDUCTION/DataFilters.h

```diff
@@ -70,24 +70,30 @@ namespace OpenMS
       EXISTS          ///< Only for META_DATA filter type, tests if meta data exists
     };
 
-    ///Representation of a peak/feature filter combining FilterType, FilterOperation and a value
+    /// Representation of a peak/feature filter combining FilterType, FilterOperation and a value (either double or String)
     struct OPENMS_DLLAPI DataFilter
```

Comment:
should this class be moved to the GUI ? It seems to be only used there

Reply:
maybe, yes.
But I can imagine having use cases in the main lib as well.
Would keep it there for now.

Reply:
We have quite some filtering algorithms in the main lib. Don't know if these classes can be married. Otherwise, having different concepts might confuse users